libs/utils/trace.py

```diff
@@ -128,14 +128,8 @@ def __init__(self, platform, data_dir, events,
                           trace_format)
         self.__computeTimeSpan()
 
-        # Minimum and Maximum x_time to use for all plots
-        self.x_min = 0
-        self.x_max = self.time_range
```

Comment:
Regarding the intialization to `self.window[]`, in order to do that we should make that window to be `(0, self.time_range)` if the user didn't specify a window and `normalize_time` is true.